### PR TITLE
Migrate noderesources and noderesourcetopology directories to structured logging

### PR DESCRIPTION
**Pull request by yharish991**
This PR  migrates noderesources and noderesourcetopology directories to structed logging

Files changed:
pkg/noderesources/allocatable.go
pkg/noderesources/resource_allocation.go
pkg/noderesourcetopology/filter.go

This pr fixes part 5 of https://github.com/kubernetes-sigs/scheduler-plugins/issues/120

### CODE REVIEW

**Comment by yharish991** [comment]
@Huang-Wei can you review this pr when you get a chance.. thanks..

**Comment by yharish991** [comment]
I signed it

File: pkg/noderesources/resource_allocation.go [hunk 1]
```diff
@@ -73,20 +73,16 @@ func (r *resourceAllocationScorer) score(
...
 		if len(pod.Spec.Volumes) >= 0 && utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) && nodeInfo.TransientInfo != nil {
-			klog.Infof(
-				"%v -> %v: %v, map of allocatable resources %v, map of requested resources %v , allocatable volumes %d, requested volumes %d, score %d",
-				pod.Name, node.Name, r.Name,
-				allocatable, requested, nodeInfo.TransientInfo.TransNodeInfo.AllocatableVolumesCount,
-				nodeInfo.TransientInfo.TransNodeInfo.RequestedVolumes,
-				score,
-			)
+			klog.InfoS("Resources, volumes and score",
+				"podName", pod.Name, "nodeName", node.Name, "resourceAllocationScorerName", r.Name,
```
Comment (Huang-Wei) [inline pkg/noderesources/resource_allocation.go:77]: nit: "resourceAllocationScorerName" -> "scorer" (applies elsewhere)

File: pkg/noderesources/resource_allocation.go [hunk 2]
```diff
@@ -73,20 +73,16 @@ func (r *resourceAllocationScorer) score(
...
+				"score", score)
 		} else {
-			klog.Infof(
-				"%v -> %v: %v, map of allocatable resources %v, map of requested resources %v ,score %d,",
-				pod.Name, node.Name, r.Name,
-				allocatable, requested, score,
-			)
-
+			klog.InfoS("Resources and score",
+				"podName", pod.Name, "nodeName", node.Name, "resourceAllocationScorerName", r.Name,
```
Comment (Huang-Wei) [inline pkg/noderesources/resource_allocation.go:83]: ditto. "scorer" should suffice.

**Review by Huang-Wei — COMMENTED** [review]
One nit. LGTM otherwise.

Could you squash the commits along with resolving the comment?

**Comment by yharish991** [comment]
@Huang-Wei PTAL :)

**Comment by yharish991** [comment]
@Huang-Wei is this ready to be merged?

**Comment by Huang-Wei** [comment]
> is this ready to be merged?

I thought I approved it...

/lgtm
/approve

**Pull request merged**